.github/workflows/test-jpf.yml

```diff
@@ -0,0 +1,41 @@
+name: Run TLC JPF Tests
+# These test take a long time to run. We only want to run them whenever the tested classes are updated.
+
+on:
+  push:
+    paths:
+      - 'tlatools/org.lamport.tlatools/src/tlc2/tool/fp/OffHeapDiskFPSet.java'
+      - 'tlatools/org.lamport.tlatools/test-verify/tlc2/tool/fp/OffHeapDiskFPSetJPFTest.java'
+      - 'tlatools/org.lamport.tlatools/src/tlc2/tool/queue/StateQueue.java'
+      - 'tlatools/org.lamport.tlatools/test-verify/tlc2/tool/queue/StateQueueJPFTest.java'
+  pull_request:
+    paths:
+      - 'tlatools/org.lamport.tlatools/src/tlc2/tool/fp/OffHeapDiskFPSet.java'
+      - 'tlatools/org.lamport.tlatools/test-verify/tlc2/tool/fp/OffHeapDiskFPSetJPFTest.java'
+      - 'tlatools/org.lamport.tlatools/src/tlc2/tool/queue/StateQueue.java'
+      - 'tlatools/org.lamport.tlatools/test-verify/tlc2/tool/queue/StateQueueJPFTest.java'
+
+jobs:
+  run-tlc-tests:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Set up JDK 11
+        uses: actions/setup-java@v1
+        with:
+          java-version: '11'
+
+      ##
+      ## Speed-up build with a cached ~/.m2/repository (300+ MB).
+      ##
+      - uses: actions/cache@v1
+        with:
+          path: ~/.m2/repository
+          key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
+          restore-keys: |
+            ${{ runner.os }}-maven-
+
+      - name: Run TLC tests
+        run: ant -f tlatools/org.lamport.tlatools/customBuild.xml -Dtest.halt=true compile compile-test test-verify
```

Comment:
Also call the `info` target to include build information in the test output (if any).

Reply:
do you want me to update [pr.yml](https://github.com/tlaplus/tlaplus/blob/fa4968f60a0a460da4b7e182330680ce93bd96aa/.github/workflows/pr.yml#L118) and [main.yml](https://github.com/tlaplus/tlaplus/blob/fa4968f60a0a460da4b7e182330680ce93bd96aa/.github/workflows/main.yml#L107) as well?